apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/ErrorReporting/ErrorReportingExceptionFilter.cs

```diff
@@ -70,15 +70,32 @@ public class ErrorReportingExceptionFilter : IExceptionFilter, IDisposable
         public static ErrorReportingExceptionFilter Create(string projectId, string serviceName, string version,
             ErrorReportingOptions options = null)
         {
-            GaxPreconditions.CheckNotNullOrEmpty(projectId, nameof(projectId));
```

Comment:
It seems like the "Cannot be null" portion of the `projectId` description is no longer valid.